deploy/helm/kubecf/assets/operations/instance_groups/brain-tests.yaml

```diff
@@ -0,0 +1,48 @@
+- path: /releases/-
+  type: replace
+  value:
+    name: brain-tests
+    url: {{ include "kubecf.releaseURLLookup" (list .Values.releases "brain-tests") }}
+    version: v0.0.5
+    stemcell: {{ include "kubecf.stemcellLookup" (list .Values.releases "brain-tests") }}
+
+- path: /instance_groups/-
+  type: replace
+  value:
+    azs:
+    - z1
+    env:
+      bosh:
+        agent:
+          settings:
+            serviceAccountName: {{ .Values.deployment_name }}-tests-brain
+    instances: 1
+    jobs:
+    - name: acceptance-tests-brain
+      properties:
+        acceptance_tests_brain:
+          domain:        ((system_domain))
+          apps_domain:   ((system_domain))
+          namespace:     {{ .Release.Namespace }}
+          org:           test-brain-org
+          password:      ((cf_admin_password))
+          space:         test-space-org
+          storage_class: {{ default "hostpath" .Values.kube.storage_class | quote }}
```

Comment:
Should we just be using the default kube storage class instead?

Reply:
IOW you are asking to change `hostpath` to `default` ?
If yes, no idea if that is ok or not. :arrow_right: @f0rmiga

Reply:
`default` won't necessarily work. We probably need to change how we deal with the default storage class on the release itself. I would expect the scripts that deal with the `storage_class` property to use it when set and to use the default storage class set in the cluster when not set. E.g., my storage classes look like this:

```
NAME                   PROVISIONER               AGE
local-path (default)   rancher.io/local-path     153m
standard               kubernetes.io/host-path   153m
```

Reply:
Addressed in commit bc7d9d062eb1daffb6cb579426c1f6ff93629fa2